### PR TITLE
delete temp shapefiles

### PR DESCRIPTION
**Pull request by pHill5136**

### CODE REVIEW

**Comment by pHill5136** [comment]
@csmoore @kgonzago please review and merge

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/Models/KMLUtils.cs [hunk 1]
```diff
@@ -61,9 +62,23 @@ class KMLUtils
...
+                    if(layer.Name==outshp)
+                    {
+                        layer1 = layer;
+                    }
+                    else if(layer.Name==nameNoExtension)
+                    {
+                        layer2 = layer;
+                    }
+                }
+                //var layer = MapView.Active.GetSelectedLayers()[0];
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/Models/KMLUtils.cs]: stray comments

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/Models/FeatureClassUtils.cs [hunk 2]
```diff
@@ -408,7 +408,9 @@ private static IReadOnlyList<string> makeValueArray (string featureClass, string
                     foreach (string extension in extensionNames)
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/Models/FeatureClassUtils.cs]: Could also call GP.Delete if you didn't want to loop through extensions

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/Models/FeatureClassUtils.cs [hunk 3]
```diff
@@ -408,7 +408,9 @@ private static IReadOnlyList<string> makeValueArray (string featureClass, string
...
                     {
                         string shapeFile = Path.Combine(connection, datasetNoExtension + extension);
-                        //File.Delete(shapeFile);
+                        string shapefileproj = Path.Combine(connection, datasetNoExtension+ "_proj" + extension);
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/Models/FeatureClassUtils.cs]: Might want to make "_proj" a constant if used elsewhere

**Pull request merged**